### PR TITLE
CON-2533-Markdown-Multiplication-Table-exercise

### PR DESCRIPTION
**Pull request by zanninso**

### CODE REVIEW

File: subjects/java/checkpoints/multiplication-table/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+## MultiplicationTable
+
+### Instructions
+
+in a file named `MultiplicationTable.java` Write a function `generate` which takes a number as a parameter, and prints the multiplication table for this number.
```
Comment (nprimo) [inline subjects/java/checkpoints/multiplication-table/README.md:5]: This sentence should start with a capital letter

Comment (nprimo) [inline subjects/java/checkpoints/multiplication-table/README.md:5]: `Write` should be lower case

**Review by nprimo — CHANGES_REQUESTED** [review]
I left some comments inline to fix some minor typos

File: subjects/java/checkpoints/multiplication-table/README.md (hunk 1 above)
Comment (zanninso) [reply subjects/java/checkpoints/multiplication-table/README.md:5]: done

Comment (zanninso) [reply subjects/java/checkpoints/multiplication-table/README.md:5]: done

**Review by nprimo — APPROVED** [review]
LGTM

**Pull request merged**